root/etc/cont-init.d/50-config

```diff
@@ -35,11 +35,19 @@ done
 
 # make our folders and links
 mkdir -p \
-	/config/{log/letsencrypt,log/fail2ban,etc/letsencrypt,fail2ban,crontabs,dns-conf,geoip2db} \
+	/config/{log/letsencrypt,log/fail2ban,etc/letsencrypt,fail2ban,crontabs,dns-conf,geoip2db,renewal-hooks} \
 	/var/run/fail2ban
 rm -rf /etc/letsencrypt
 ln -s /config/etc/letsencrypt /etc/letsencrypt
 
+rm -rf /config/etc/letsencrypt/renewal-hooks
```

Comment:
This rm with sub directories covers the update case. As certbot creates the directories renewal-hooks{pre,post,deploy} when it does not exist.